source/includes/_all-get-apis.md

```diff
@@ -6,13 +6,13 @@ The following GET APIs are used to fetch the data from the respective tables.
 
 All data from the Attribute Type table will be returned from the endpoint below. You can optionally fetch specific data by including Attribute ID’s and Attribute Types. The Attribute Type endpoint supports pagination.
 
-> Example GET Endpoint Without Pagination: 
+> Example Of AttributeType GET Endpoint Without Pagination: 
 
 ```
-https: // *actsapi.intelex.com/API-develop/v1/AttributeType
+https: // {tenant}actsapi.intelex.com/API-develop/v1/AttributeType
```

Comment:
should there be . between {tenant} and actsapi?

Reply:
It will be {tenant}.actsapi for example : Company or Client Name is : Fluke then URL would be something like this https://fluke.actsapi.intelex.com/API-develop/v1/AttributeType .